app/assets/stylesheets/finder_frontend.scss

```diff
@@ -385,3 +385,7 @@ mark {
   margin-bottom: govuk-spacing(7);
   padding-bottom: govuk-spacing(0);
 }
+
+.govuk-radios--small {
```

Comment:
Making changes like this that affect components inside apps are against our component principles - if a component needs to appear differently, that change needs to be made on the component itself, either as the default behaviour or as an option.

Reply:
Related to this, why is everything indented by 12px, instead of making it flush with the left edge of the column? (which would align everything with the left edge of the search box and make this change unnecessary)

Reply:
the design change here doesn't need to be on the component itself, so no need for the option. If components allowed passing through custom classes, that would be useful.
We're going to wrap these radios in a `facet-container` and apply the padding there

Reply:
That's a better option, thanks.

Why is everything indented though?

Reply:
because of the hover effect on radios and checkboxes. padding on option-select and expander that was there before hasn't changed

Reply:
It's done, we've now got the radio buttons inside a container that's being used to position them correctly, rather than modifying the component itself.